### PR TITLE
(oSPARC connection) Adding stand-alone file option

### PR DESCRIPTION
**Pull request by ignapas**
# Description

* Adding a permanent choice for opening the file stand-alone in oSPARC. The file opens in a new study in a lonely file picker.
* Fixes broken link

Notes
* Waiting to check some small issues before making it ready for review
* Possibly a new icon should be used for oSPARC-related actions in the file table **DONE**

## Type of change

- [x] New feature (non-breaking change which adds functionality)

# How Has This Been Tested?

Manually

# Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have utilized components from the Design System Library where applicable
- [ ] I have added unit tests that prove my fix is effective or that my feature works

### CODE REVIEW

**Review by pcrespov — COMMENTED** [review]
pls reassign when ready to review. htx

**Comment by pcrespov** [comment]
@ignapas can we change "Open Stand-alone" by something that suggest "export"? e.g. "Export File to osparc"  or "Use File in Osparc"?
image

**Review by pcrespov — APPROVED** [review]

File: components/FilesTable/FilesTable.vue [hunk 1]
```diff
@@ -192,7 +192,6 @@
                 </sparc-tooltip>
               </div>
               <div
-                v-if="hasOsparcViewer(scope)"
```
Comment (egauzens) [inline components/FilesTable/FilesTable.vue:195]: So we want to show this always, even if there is no viewer available?

Comment (ignapas) [reply components/FilesTable/FilesTable.vue:195]: Yes, now it is possible to open every file type for custom processing in oSparc

**Pull request merged**